exercises/concept/lasagna/.docs/instructions.md

```diff
@@ -2,9 +2,18 @@
 
 In this exercise you're going to write some code to help you cook a brilliant lasagna from your favorite cooking book.
 
-You have three tasks, all related to the time spent cooking the lasagna.
+You have four tasks, all related to the time spent cooking the lasagna.
 
-## 1. Calculate the preparation time in minutes
+## 1. 
+
+Define the `expected_bake_time` constant that returns how many minutes the lasagna should bake in the oven. According to the cooking book, lasagna needs to be in the oven for a total of 60 minutes.
+
+```julia-repl
+julia> expected_bake_time
+60
+```
+
+## 2. Calculate the preparation time in minutes
 
 Define the `preptime` function that takes the number of layers you added to the lasagna as an argument and returns how many minutes you spent preparing the lasagna, assuming each layer takes you 2 minutes to prepare.
```

Comment:
Shouldn't this be `prep_time` globally to match the other functions?

Reply:
Good catch. It's old code that I didn't read carefully enough.

Reply:
For real consistency, I went with `preparation_time` in the revisions I'll push next.

Reply:
This changes touches most of the files. A quick `grep -r preptime` reassures me that I caught them all.

I caught a few other errors while doing this, especially in `hints.md`.